amgut/db/patches/0009.sql

```diff
@@ -0,0 +1,29 @@
+-- Add created_on timestamp for handout kits
+ALTER TABLE ag.ag_handout_kits ADD created_on timestamp DEFAULT current_timestamp NOT NULL;
+
+-- Create new handout_barcode table
+CREATE TABLE ag.handout_barcode ( 
```

Comment:
while the naming structure is redundant and its use of plurals is not great, but it is better to be consistent -- could this be changed to `ag_handout_barcodes`?